radixsort.c

```diff
@@ -114,6 +114,10 @@ static void _bisect_radix(void * r, const void * r1, const void * r2, size_t rsi
     const unsigned char * u2 = r2;
     unsigned char * u = r;
     unsigned int carry = 0;
+    if(dir > 0) {
+        u1 += rsize - 1;
+        u2 += rsize - 1;
+    }
```

Comment:
This didn't bite us, because it only affected be, but the tests (and use cases) only covered le?

Reply:
Right!